src/main/zip/info.xml

```diff
@@ -184,5 +184,8 @@
     <release-note plugin-version="22">
         RFE 142129 - Added Deployment-Policy-variables field to the Import (Deployment Policy Object) step.
     </release-note>
+    <release-note plugin-version="23">
+        RFE 142314  - Need ability to export more than a single class or object.
```

Comment:
Going forward, I would suggest a release note that better aligns to the plugin change, rather than the RFE request.

Something like ... `RFE 142314 - Added new Export Objects steps that enables the export of one or more objects.`